config_project_settings

```diff
@@ -0,0 +1,95 @@
+---
+title: ''
```

Comment:
What title you suggest?
Also, can you suggest a description (1 sentence or phrase)

Reply:
Oh, sorry, I now see this below.